client/dashboard/app/routes/domain-routes.ts

```diff
@@ -119,8 +119,10 @@ export const domainForwardingsRoute = createRoute( {
 export const domainForwardingAddRoute = createRoute( {
 	getParentRoute: () => domainRoute,
 	path: 'forwarding/add',
+	loader: ( { params: { domainName } } ) =>
+		queryClient.ensureQueryData( domainForwardingQuery( domainName ) ),
 } ).lazy( () =>
-	import( '../../sites/domains/placeholder' ).then( ( d ) =>
+	import( '../../domains/domain-forwarding/add' ).then( ( d ) =>
```

Comment:
I think we should consolidate the route and directory patterns here.

For example, in the [glue records PR](https://github.com/Automattic/wp-calypso/pull/105184), we use the following paths:

- `glue-records`
- `glue-records/add`
- `glue-records/edit/X`

While here, we're using:

- `forwardings`
- `forwarding/add`
- `forwarding/edit/X`

I think we should pick one pattern and stick 😄 my recommendations would be always to use plurals, since it's what we've been using for domain and site overview paths.

With this, the file structure can be:

- `domain-forwardings/index.tsx`
- `domain-forwardings/add.tsx`
- `domain-forwardings/edit.tsx`

Open for suggestion. 😄